index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <!-- Header section -->
+    <header class="header">
+      <nav class="nav-bar d-flex">
+        <a href="#"><h3 class="logo">Ha Ell</h3></a>
+        <img src="./image_icons/Icon-Menu.svg" alt="bar icon"/>
+      </nav>
+    </header>
+    <!-- End of Header section -->
+    <!-- Headline section -->
+    <section class="headline-section">
+      <div class="headline-container d-flex">
+        <div class="headline-title">
+          <h1>hey there. <br><br> I am Hamza</h1>
+        </div>
```

Comment:
- [ ] Optional
As it was requested by the previous reviewers, please work on it before you merge
![hi](https://user-images.githubusercontent.com/55635977/125056979-0c933180-e0a9-11eb-8965-3fc9ce6a1952.png)
## Change your font-size and font-weight